lib/factory_girl/definition_proxy.rb

```diff
@@ -94,6 +95,17 @@ def transient(&block)
     #
     # are equivalent.
     def method_missing(name, *args, &block)
+      # If methods are called from debugger, original method should be called
+      if defined?(caller_locations) &&
+        (['eval', 'process_commands', 'build_compact_name','build_compact_value_attr', 'print_element'].include?(caller_locations[1].base_label) ||
```

Comment:
Line is too long. [147/80]<br>Align the operands of a condition in an `if` statement spanning multiple lines.<br>Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.